android/modules/ui/src/java/ti/modules/titanium/ui/widget/tableview/TiTableView.java

```diff
@@ -68,6 +72,45 @@ public TiTableView(TableViewProxy proxy)
 		this.recyclerView.setBackgroundColor(Color.TRANSPARENT);
 		this.recyclerView.setLayoutManager(new LinearLayoutManager(getContext()));
 
+		// Add listener to fire scroll events.
+		this.recyclerView.addOnScrollListener(new RecyclerView.OnScrollListener()
+		{
+			@Override
+			public void onScrollStateChanged(@NonNull RecyclerView recyclerView, int newState)
+			{
+				super.onScrollStateChanged(recyclerView, newState);
+
+				if (newState == RecyclerView.SCROLL_STATE_IDLE) {
+					final KrollDict payload = generateScrollPayload();
+					final TiNestedRecyclerView nestedRecyclerView = getRecyclerView();
+
+					// Obtain last touch position for `scrollend` event.
+					final TiDimension xDimension =
+						new TiDimension(nestedRecyclerView.getLastTouchX(), TiDimension.TYPE_WIDTH);
+					final TiDimension yDimension =
+						new TiDimension(nestedRecyclerView.getLastTouchY(), TiDimension.TYPE_HEIGHT);
+					payload.put(TiC.EVENT_PROPERTY_X, xDimension.getAsDefault(nestedRecyclerView));
+					payload.put(TiC.EVENT_PROPERTY_Y, yDimension.getAsDefault(nestedRecyclerView));
+
+					proxy.fireSyncEvent(TiC.EVENT_SCROLLEND, payload);
+				}
```

Comment:
If I set `TableView.scrollable` to `false`, then the "scrollend" fires when it shouldn't when I drag my finger over the `TableView`. We should probably add an `isScrolling` member variable and guard against this like in `ListView`.